cf-agent/retcode.c

```diff
@@ -44,8 +44,15 @@ bool VerifyCommandRetcode(EvalContext *ctx, int retcode, const Attributes *a, co
         xsnprintf(retcodeStr, sizeof(retcodeStr), "%d", retcode);
 
         LogLevel info_or_verbose = LOG_LEVEL_INFO;
+
+        // inform constraint is only for commands promises,
+        // a->inform is actually false for other promise types, so
+        // checking the promise type here is important:
         if (StringSafeEqual("commands", pp->parent_promise_type->name) && (!a->inform))
         {
+            // for commands promises which don't make changes to the system,
+            // you can use this to make the log messages verbose:
+            // inform => "false";
```

Comment:
you can use `inform => "false"` to make the log messages verbose?

Reply:
Yes